docker-compose-arm64.yml

```diff
@@ -54,6 +54,9 @@ services:
       SECRETSTORE_HOST: edgex-vault
       SECRETSTORE_PORT: '8200'
       SERVICE_HOST: edgex-app-rules-engine
+      SPIFFE_ENDPOINTSOCKET: /tmp/edgex/secrets/spiffe/public/api.sock
+      SPIFFE_TRUSTBUNDLE_PATH: /tmp/edgex/secrets/spiffe/trust/bundle
+      SPIFFE_TRUSTDOMAIN: edgexfoundry.org
```

Comment:
@jim-wang-intel why am I getting these in my PR. Did you forget to do a `make build` on a previous PR? Please open new PR with these changes. I will rebase mine once yours is merged.

Reply:
fixed by PR https://github.com/edgexfoundry/edgex-compose/pull/230